modules/ROOT/pages/deployment/services/s-list/storage-users.adoc

```diff
@@ -60,66 +69,80 @@ ocis storage-users uploads <command>
 
-Command to identify incomplete uploads::
-+
---
 [source,bash]
 ----
-ocis storage-users uploads list
+ocis storage-users trash-bin <command>
 ----
 
 [source,plaintext]
 ----
-Incomplete uploads:
- - 455bd640-cd08-46e8-a5a0-9304908bd40a (file_example_PPT_1MB.ppt, Size: 1028608, Expires: 2022-08-17T12:35:34+02:00)
+COMMANDS:
+   purge-expired  Purge expired trash-bin items
+   list           Print a list of all trash-bin items of a space.
+   restore-all    Restore all trash-bin items for a space.
+   restore        Restore a trash-bin item by ID.
 ----
---
 
-{empty}
+==== Purge Expired
 
-Command to clear expired uploads::
-+
---
-[source,bash]
-----
-ocis storage-users uploads clean
-----
+Purge all expired items from the trash-bin.
 
 [source,plaintext]
 ----
-Cleaned uploads:
-- 455bd640-cd08-46e8-a5a0-9304908bd40a (Filename: file_example_PPT_1MB.ppt, Size: 1028608, Expires: 2022-08-17T12:35:34+02:00)
+ocis storage-users trash-bin purge-expired
 ----
---
 
-=== Purge Expired Space Trash-Bin Items
+The behaviour of the `purge-expired` command can be configured by using the following environment variables.
+
+* `STORAGE_USERS_PURGE_TRASH_BIN_USER_ID` +
+Used to obtain space trash-bin information and takes the system admin user as the default which is the `OCIS_ADMIN_USER_ID` but can be set individually. It should be noted, that the `OCIS_ADMIN_USER_ID` is only assigned automatically when using the single binary deployment and must be manually assigned in all other deployments. The command only considers spaces to which the assigned user has access and delete permission.
+
+* `STORAGE_USERS_PURGE_TRASH_BIN_PERSONAL_DELETE_BEFORE` +
+Has a default value of `30 days`, which means the command will delete all files older than `30 days`. The value is human-readable, valid values are `24h`, `60m`, `60s` etc. `0` is equivalent to disable and prevents the deletion of `personal space` trash-bin files.
+
+* `STORAGE_USERS_PURGE_TRASH_BIN_PROJECT_DELETE_BEFORE` +
+Has a default value of `30 days`, which means the command will delete all files older than `30 days`. The value is human-readable, valid values are `24h`, `60m`, `60s` etc. `0` is equivalent to disable and prevents the deletion of `project space` trash-bin files.
+
+==== List and Restore Trash-Bins Items
 
-This command is about purging old trash-bin items of `project` spaces (spaces that have been created manually) and `personal` spaces.
+The variable `STORAGE_USERS_CLI_MAX_ATTEMPTS_RENAME_FILE` defines a maximum number of attempts to rename a file when the admin restores the file with the CLI option `--option keep-both` to existing destination with the same name.
```

Comment:
```suggestion
The variable `STORAGE_USERS_CLI_MAX_ATTEMPTS_RENAME_FILE` defines a maximum number of attempts to rename a file when the admin restores the file with the CLI option `--option keep-both` to an existing destination with the same name.
```

Is it possible to provide some reasons why someone might want to set this, what might be a "reasonable" value, and what might be the next steps if a CLI user has this failing after the retries.